modules/electrophysiology_browser/help/view_session.md

```diff
@@ -0,0 +1,15 @@
+# View Session
+
+This page displays all available information collected for an individual recording session.
+
+Summary information about the candidate, the recording session and the hardware used is displayed.
+
+Recording files can be downloaded individually or as a compressed folder (*All Files* download):
+
+- EEG: the file containing the session recording data.    
+- Electrode info (tsv): contains electrode locations.    
+- Channels info (tsv): channel status and filter settings.    
+- Events (tsv): events recorded during the session.   
+
+The compressed folder’s structure follows the BIDS standard for the EEG modality. More information about the BIDS standard can be found through their [website](https://bids.neuroimaging.io/).   
```

Comment:
```suggestion
```
Moving the BIDS context earlier -- because the "Events" download doesn't make sense without mentioning BIDS first